packages/plugin/tests/__snapshots__/examples.spec.ts.snap

```diff
@@ -3,48 +3,48 @@
 exports[`Examples should work on \`.graphql\` files 1`] = `
 Array [
   Object {
-    "filePath": "examples/basic/fragment.graphql",
-    "messages": Array [
+    filePath: examples/basic/fragment.graphql,
+    messages: Array [
       Object {
-        "message": "Fragment named \\"Test\\" already defined in:
```

Comment:
I added package https://www.npmjs.com/package/jest-snapshot-serializer-raw for reducing escapes in the snapshot file